### PR TITLE
TMA-775: smart attribute polling

### PR DESCRIPTION
**Pull request by kubamahnert**

### CODE REVIEW

**Comment by kubamahnert** [comment]
extended test

File: lib/gooddata/models/user_filters/user_filter_builder.rb [hunk 1]
```diff
@@ -140,11 +140,10 @@ def self.verify_existing_users(filters, options = {})
 
     def self.create_label_cache(result, options = {})
       project = options[:project]
-      project_labels = project.labels
 
       result.reduce({}) do |a, e|
         e[:filters].map do |filter|
-          a[filter[:label]] = project_labels.find { |l| (l.identifier == filter[:label]) || (l.uri == filter[:label]) } unless a.key?(filter[:label])
+          a[filter[:label]] = project.labels(filter[:label]) unless a.key?(filter[:label])
```
Comment (yenkins) [inline lib/gooddata/models/user_filters/user_filter_builder.rb]: Calls 'filter[:label]' 3 times (DuplicateMethodCall)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/750/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/611/ : SUCCESS in 2m 08s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/872/ : SUCCESS in 2m 03s

**Comment by panjan** [comment]
ok to merge

**Pull request merged**